simplq/src/components/pages/Join/Form.jsx

```diff
@@ -77,7 +82,17 @@ export function JoinQueueForm(props) {
                 inputStyle={{
                   width: '100%',
                 }}
-                isValid={() => (invalidContact ? 'Phone number is not valid' : true)}
+                isValid={(inputNumber, country) => {
+                  var arr=[];
+                  arr = inputNumber.split(country.dialCode);
```

Comment:
Use const instead of var.

Reply:
The first var arr=[] can be removed safely and const arr = inputNumber... can be used I believe.